pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -239,6 +240,33 @@ func (m *ManagerImpl) Start(activePods ActivePodsFunc, sourcesReady config.Sourc
 	return nil
 }
 
+// GetWatcherCallback returns callback function to be registered with plugin watcher
+func (m *ManagerImpl) GetWatcherCallback() watcher.RegisterCallbackFn {
+	return func(name string, endpoint string, versions []string, sockPath string) (chan bool, error) {
```

Comment:
Do you think whether we should create DEPRECATION file under m.socketDir before returning the callback function as we discussed in https://github.com/kubernetes/kubernetes/pull/58755#discussion_r199306009? This way, people can start testing the migration with pluginwatcher enabled and we can extend the device plugin e2e_node test and e2e test to start getting test coverage.